### ISSUE TITLE
displayname not show on MentionArrayAdapter

### ISSUE THREAD

**Issue by jasoncheng** [hendraanggrian/socialview#53]
as title, please help

```
doAsync {
            val users = AppDatabase.getInstance().userDao().all()
            val adapter = MentionArrayAdapter<Mention>(context!!)
            uiThread {
                users?.forEach { that ->
                    dLog("User ${that.getEmail()} ${that.name}")
                    adapter.add(Mention(that.getEmail(), "${that.name}", that.profile_image_url_large))
                }
                et_text.mentionAdapter = adapter
            }
}
```
![15574](https://user-images.githubusercontent.com/540463/49428411-f336f400-f7e0-11e8-9091-054e2012f3cd.jpg)

BTW, thanks for this awesome library 👍

**Comment by hendraanggrian** [comment]
`Mention.displayname` is a nullable property, make sure your user instance and `User.text` is not null.

**Comment by jasoncheng** [comment]
even i change it as below, the displayname still not shown
`Mention(that.getEmail(), "this is test 1234", that.profile_image_url_large)`

implementation "com.hendraanggrian.appcompat:socialview:0.1"
implementation "com.hendraanggrian.appcompat:socialview-commons:0.1"

**Comment by hendraanggrian** [comment]
Sorry for delayed response. I'm aware of this issue and will be fixed in 0.2. Thanks for pointing this out.
